bitcoin_executor/sources/block.move

```diff
@@ -0,0 +1,48 @@
+module bitcoin_executor::block;
+use bitcoin_executor::block_header::{BlockHeader, Self};
+use bitcoin_executor::tx::{Transaction, Self};
+use bitcoin_executor::reader;
+
```

Comment:
formatting

Reply:
I don't have the fmt for the sui move yet. Can we do this later?